documentation/modules/ROOT/partials/modules/all-connectors/ref-connector-configuration-database-history-properties.adoc

```diff
@@ -0,0 +1,78 @@
+{prodname} provides a set of `database.history.*` properties that control how the connector processes events that it reads from the schema history topic.
+These `database.history` properties specify how the {context} connector processes the event data that it reads from the database history.
+
+The following table describes the `database.history` properties for configuring the {prodname} {context} connector.
+
+.Connector database history configuration properties
+[cols="33%a,17%a,50%a",options="header",subs="+attributes"]
+|===
+|Property |Default |Description
+|[[{context}-property-database-history-kafka-topic]]<<{context}-property-database-history-kafka-topic, `+database.history.kafka.topic+`>>
+|
+|The full name of the Kafka topic where the connector stores the database schema history.
+
+|[[{context}-property-database-history-kafka-bootstrap-servers]]<<{context}-property-database-history-kafka-bootstrap-servers, `+database.history.kafka.bootstrap.servers+`>>
+|
+|A list of host/port pairs that the connector uses for establishing an initial connection to the Kafka cluster. This connection is used for retrieving database schema history previously stored by the connector, and for writing each DDL statement read from the source database. Each pair should point to the same Kafka cluster used by the Kafka Connect process.
```

Comment:
```suggestion
|A list of host/port pairs that the connector uses for establishing an initial connection to the Kafka cluster. This connection is used for retrieving the database schema history previously stored by the connector, and for writing each DDL statement read from the source database. Each pair should point to the same Kafka cluster used by the Kafka Connect process.
```

Reply:
It's in db2 (L2892), mysql (L2174), and sqlserver (L2006) docs in the `master` branch. Should it not be included?

Reply:
I'm not sure what you're asking? I just found that the "the" is missing there (see darker green marked area).

Reply:
@gunnarmorling Sorry, I responded in the wrong place. I was answering your question about where did this one come from: https://github.com/debezium/debezium/pull/2455#discussion_r657830855